vueapp/src/components/Waiting.vue

```diff
@@ -86,6 +92,19 @@
   margin-left: 5px;
   margin-right: 5px;
 }
+
+.flip-list-move {
+  transition: transform 0.5s;
+}
+
+.no-move {
+  transition: transform 0s;
+}
```

Comment:
What are these for? I wasn't able to find them in `vuedraggable`'s docs

Reply:
Looks like left overs from some experiments